apps/src/gamelab/AnimationTab/NewListItem.jsx

```diff
@@ -52,10 +56,17 @@ var NewListItem = React.createClass({
   },
 
   render: function () {
+    var dynamicStyles = {
+      dottedBorder: {
+        backgroundColor: Radium.getState(this.state, 'main', ':hover')
+            ? color.lightest_purple : undefined
+      }
+    };
```

Comment:
I'm cool with that.  I feel like they express different things; in yours, there's a bundle of styles `dottedBorderHovered` that we want to apply when 'main' (the root element) is hovered.  In mine we create a set of dynamic styles that might all depend on different properties.  With just the one dynamic style it's neither here nor there I suppose.

Do you like that approach because it puts the hover style close to the other styles for the element?